pvlib/iotools/tmy.py

```diff
@@ -519,3 +519,179 @@ def _read_tmy2(string, columns, hdr_columns, fname):
+    direct_normal_illum                 Avg. direct normal illuminance recv'd during the 60 minutes prior to timestamp, lx
+    diffuse_horizontal_illum            Avg. horizontal diffuse illuminance recv'd during the 60 minutes prior to timestamp, lx
+    zenith_luminance                    Avg. luminance at the sky's zenith during the 60 minutes prior to timestamp, cd/m^2
+    wind_direction                      Wind direction at time indicated, degrees from north (360 = north; 0 = undefined,calm)
+    wind_speed                          Wind speed at the time indicated, meter/second
+    total_sky_cover                     Amount of sky dome covered by clouds or obscuring phenonema at time stamp, tenths of sky
+    opaque_sky_cover                    Amount of sky dome covered by clouds or obscuring phenonema that prevent observing the sky at time stamp, tenths of sky
+    visibility                          Horizontal visibility at the time indicated, km
+    ceiling_height                      Height of cloud base above local terrain (7777=unlimited), meter
+    present_weather_observation         Indicator for remaining fields: If 0, then the observed weather codes are taken from the following field. If 9, then “missing” weather is assumed. Since the primary use of these fields (Present Weather Observation and Present Weather Codes) is for rain/wet surfaces, a missing observation field or a missing weather code implies no rain.
+    present_weather_codes               Present weather code, see [1], chapter 2.9.1.28
+    precipitable_water                  Total precipitable water contained in a column of unit cross section from earth to top of atmosphere, cm
+    aerosol_otpical_depth               The broadband aerosol optical depth per unit of air mass due to extinction by aerosol component of atmosphere, unitless
+    snow_depth                          Snow depth in centimeters on the day indicated, (999 = missing data)
+    days_since_last_snowfall            Number of days since last snowfall (maximum value of 88, where 88 = 88 or greater days; 99 = missing data)
+    albedo                              The ratio of reflected solar irradiance to global horizontal irradiance, unitless
+    liquid_precipitation_depth          The amount of liquid precipitation observed at indicated time for the period indicated in the liquid precipitation quantity field, millimeter
+    liquid_precipitation_quantity       The period of accumulation for the liquid precipitation depth field, hour
+    =============================       ======================================================================================================================================================
+
+    References
+    ----------
+
+    [1] EnergyPlus documentation, Auxiliary Programs
+    https://energyplus.net/documentation.
+    '''
+    
+    if filename is None:
+        try:
+            filename = _interactive_load()
```

Comment:
Does anyone use the interactive load for tmy files? I'd rather deprecate that feature in tmy than continue it here.

Reply:
Anticipating that the answer will be no, I have omitted this option in the new `read_epw` function